### ISSUE TITLE
docs: Document how to bootstrap Indico on the CLI

### ISSUE THREAD

**Issue by paulmenzel** [indico/indico#5392]
Accessing Indico for the first time, the path `/bootstrap` is accessed to set up the admin account, fill in the affiliation, and to send the installation data to the Indico hub.

It’d be great, if it were documented how to do that on the CLI.

**Comment by ThiefMaster** [comment]
There's no CLI for this. Just wondering, what's your usecase for it? While I understand wanting to automate setting up an indico instance infrastructure-wise, automating the initial account creation seems rather uncommon...

**Comment by paulmenzel** [comment]
In my case, I set everything up, so the server is accessible from outside, so anybody could have accessed that page in the small time frame, and set up the admin account. I was also a little surprised, as I expecting the setup wizard did everything, and the bootstrap page is not mentioned anywhere in the documentation.

Sharing the setup scripts, people might run them, do something else (and forget about it), and then the time windows for others accessing this page might be even quite large.

**Comment by ThiefMaster** [comment]
Good point. Even though the risk is very limited: Being an indico admin doesn't give you any ways to execute nasty code on the server and there's no sensitive data in an empty instance. And the real admin creating their account would probably be surprised that they don't have admin permissions and realize something is fishy.

Still, we could probably improve this a bit e.g. by prompting for a code (displayed during the setup wizard) on the bootstrap page, or by letting the user create the initial account on the CLI as you suggested.

**Comment by paulmenzel** [comment]
Configuring a special access rule for the path `/bootstrap` in the HTTPD server would also be an option, though the allowed IP range would need to be manually configured.

**Comment by amandahla** [comment]
Any news on that one?
